osu.Game/Screens/Ranking/Expanded/StarRatingDisplay.cs

```diff
@@ -107,9 +107,7 @@ private void updateDisplay()
             string fractionPart = starRatingParts[1];
             string separator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
 
-            var rating = Current.Value.DifficultyRating;
-
-            background.Colour = colours.ForDifficultyRating(rating, true);
+            background.Colour = colours.ForStarDifficulty(Current.Value.Stars);
```

Comment:
Is pitch black above 8 stars. See wedge on screenshot below.

![osu_2021-08-03_23-22-56](https://user-images.githubusercontent.com/20418176/128088144-0447cc18-46d8-45cd-890f-42250cdb8a21.jpg)

Text should be orange-1 above 6.50* ([reference](https://github.com/ppy/osu-web/pull/7855#issuecomment-881272399))

Reply:
Good catch, thought it's the case already but it appears not.